cassandra-analytics-core/src/test/java/org/apache/cassandra/spark/bulkwriter/BulkSparkConfTest.java

```diff
@@ -275,6 +277,70 @@ void testSidecarContactPoints()
         .isNull();
     }
 
+    @Test
+    void testReadCoordinatedWriteConfFails()
```

Comment:
Maybe let's have a test with an invalid JSON representation for the `WriterOptions.COORDINATED_WRITE_CONF` option?

Reply:
I believe the test case is already there.